### PR TITLE
added Pursuit40 PCB for Panc40

### PR DESCRIPTION
**Pull request by npspears**
Pursuit40 is another PCB option for the Panc40 that was sold on Panc.co/store

## Description

PCB addition

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [  ] My change requires a change to the documentation.
- [  ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [  ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/pursuit40/info.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Pursuit40", 
+    "url": "https://panc.co/store/ols/products/panc40-keyboard", 
+    "maintainer": "nasp", 
+    "width": 12, 
+    "height": 4, 
+    "layouts": {
+        "default": {
```
Comment (zvecr) [inline keyboards/pursuit40/info.json:8]:
```suggestion
        "LAYOUT_default": {
```

File: keyboards/pursuit40/rules.mk [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+UNICODE_ENABLE = yes        # Unicode
+
```
Comment (zvecr) [inline keyboards/pursuit40/rules.mk:24]: shouldnt be enabled at the keyboard level
```suggestion
```

File: keyboards/pursuit40/rules.mk [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
```
Comment (zvecr) [inline keyboards/pursuit40/rules.mk:14]:
```suggestion
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by npspears — COMMENTED** [review]
changes added to batch

**Review by drashna — APPROVED** [review]

File: keyboards/pursuit40/readme.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Pursuit40
+
+![Pursuit40](https://i.imgur.com/Ay3k75t.jpg)
```
Comment (fauxpark) [inline keyboards/pursuit40/readme.md:3]:
```suggestion
![Pursuit40](https://i.imgur.com/Ay3k75tl.jpg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/pursuit40/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+     * | Tab        |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Del  |
+     * |------------+------+------+------+------+-------------+------+------+------+------+------|
+     * | CTL & ESC  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |  "   |
+     * |------------+------+------+------+------+------|------+------+------+------+------+------|
+     * | Shift      |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |Enter |
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+------|
+     * | PRINT   |  OS   |  Alt  |  Layer  |    Space & Layer     |   [     |   ]    |    CAPS   |
+     * `-----------------------------------------------------------------------------------------'
+     */
+    [0] = LAYOUT_default(
```
Comment (drashna) [inline keyboards/pursuit40/keymaps/via/keymap.c:37]: This looks to have one more full line than the default, and the layout macro, and is erroring out due to this.

Comment (npspears) [reply keyboards/pursuit40/keymaps/via/keymap.c:36]: Correct. Extra row deleted 👍

File: keyboards/pursuit40/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+---|
+     * |     RESET    |           |          |         |         |         |        |         |
+     * `---------------------------------------------------------------------------------------'
+     */
+    [2] = LAYOUT_default(
+        _______, _______, _______, KC_UP,   _______, _______, _______, KC_UNDS, _______, KC_LBRC, KC_RBRC, _______,
+        _______, _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______, KC_MINS, _______, KC_LCBR, KC_RCBR, _______,
+        KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,
+        RESET,   _______, _______,          _______,                   _______,          _______, _______, _______
+    )
```
Comment (fauxpark) [inline keyboards/pursuit40/keymaps/via/keymap.c:79]: Need one more empty layer here.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/pursuit40/keymaps/via/keymap.c (hunk 6 above)
Comment (npspears) [reply keyboards/pursuit40/keymaps/via/keymap.c]: empty layer added

File: keyboards/pursuit40/keymaps/via/keymap.c [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+     * |      |     |      |      |      |      |      |     |     |      |      |      |      |
+     * |------------+------+------+------+------+-------------+------+------+------+------+----|
+     * |      |     |      |      |      |      |      |     |     |      |      |      |      |
+     * |------------+------+------+------+------+------|------+------+------+------+------+----|
+     * |      |     |      |      |      |      |      |     |     |      |      |      |      |
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+----|
+     * |      |     |      |      |      |      |      |     |     |      |      |      |      |
+     * `---------------------------------------------------------------------------------------'
+     */
+    [3] = LAYOUT_default(
```
Comment (drashna) [inline keyboards/pursuit40/keymaps/via/keymap.c:93]: Looks like this layer has too many items.

Comment (npspears) [reply keyboards/pursuit40/keymaps/via/keymap.c:93]: should be fixed now

**Review by fauxpark — APPROVED** [review]

File: keyboards/pursuit40/readme.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+# Pursuit40
+
+![Pursuit40](https://i.imgur.com/Ay3k75tl.jpg)
+
+The Pursuit40 PCB is a modified layout Panc40 by Alex Panc. It's similar to traditional ortho but the bottom row keys are larger than 1u. 
+
+* Keyboard Maintainer: [Nasp](https://github.com/npspears)
+* Hardware Supported: Panc40
+* Hardware Availability: https://panc.co/store/ols/products/panc40-keyboard
```
Comment (noroadsleft) [inline keyboards/pursuit40/readme.md:9]: This link is dead; should probably be:

```suggestion
* Hardware Availability: https://panc.co/store/ols/products/panc40-keyboard-pnc-kyb1
```

File: keyboards/pursuit40/pursuit40.h [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define LAYOUT_default( \
```
Comment (noroadsleft) [inline keyboards/pursuit40/pursuit40.h:23]: This macro seems to use every switch position that is present on the PCB.

```suggestion
#define LAYOUT_all( \
```

File: keyboards/pursuit40/info.json [hunk 10]
```diff
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "Pursuit40", 
+    "url": "https://panc.co/store/ols/products/panc40-keyboard", 
+    "maintainer": "nasp", 
+    "width": 12, 
+    "height": 4, 
+    "layouts": {
+        "LAYOUT_default": {
```
Comment (noroadsleft) [inline keyboards/pursuit40/info.json:8]:
```suggestion
        "LAYOUT_all": {
```

File: keyboards/pursuit40/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+     * | Tab        |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Del  |
+     * |------------+------+------+------+------+-------------+------+------+------+------+------|
+     * | CTL & ESC  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |  "   |
+     * |------------+------+------+------+------+------|------+------+------+------+------+------|
+     * | Shift      |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |Enter |
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+------|
+     * | PRINT   |  OS   |  Alt  |  Layer  |    Space & Layer     |   [     |   ]    |    CAPS   |
+     * `-----------------------------------------------------------------------------------------'
+     */
+    [0] = LAYOUT_default(
```
Comment (noroadsleft) [inline keyboards/pursuit40/keymaps/default/keymap.c:36]:
```suggestion
    [0] = LAYOUT_all(
```

File: keyboards/pursuit40/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+     * |   ~    | !    |  @   |  #  |  $   |  %   |  | ^  |  &   |  *  |  (    |  )  | BCKSPC  |
+     * |------------+------+------+------+------+-------------+------+------+------+------+----|
+     * |   \    |    1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  |   /   |
+     * |------------+------+------+------+------+------|------+------+------+------+------+----|
+     * |     |      |      |      |      |      |      |   +   |   =   |      |        |       |
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+----|
+     * |   ESC  |  CTRL-ALT-DEL   |   TASK   |         |        |    '|'    |    `    |        |
+     * `---------------------------------------------------------------------------------------'
+     */
+    [1] = LAYOUT_default(
```
Comment (noroadsleft) [inline keyboards/pursuit40/keymaps/default/keymap.c:55]:
```suggestion
    [1] = LAYOUT_all(
```

File: keyboards/pursuit40/keymaps/default/keymap.c [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+     * |       |      |      |  UP  |       |      |     |   _   |     |   [   |  ]    |      |
+     * |------------+------+------+------+------+-------------+------+------+------+------+---|
+     * |       |     | LEFT | DOWN  | RIGHT |     |      |   -  |      |   [   |   ]   |      |
+     * |------------+------+------+------+------+-----+-----+------+------+------+------+-----|
+     * |   F1  | F2 |  F3  |   F4  | F5    | F6  | F7   | F8   | F9    | F10  | F11   | F12   |
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+---|
+     * |     RESET    |           |          |         |         |         |        |         |
+     * `---------------------------------------------------------------------------------------'
+     */
+    [2] = LAYOUT_default(
```
Comment (noroadsleft) [inline keyboards/pursuit40/keymaps/default/keymap.c:74]:
```suggestion
    [2] = LAYOUT_all(
```

File: keyboards/pursuit40/keymaps/via/keymap.c [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+     * | Tab        |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Del  |
+     * |------------+------+------+------+------+-------------+------+------+------+------+------|
+     * | CTL & ESC  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |  "   |
+     * |------------+------+------+------+------+------|------+------+------+------+------+------|
+     * | Shift      |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |Enter |
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+------|
+     * | PRINT   |  OS   |  Alt  |  Layer  |    Space & Layer     |   [     |   ]    |    CAPS   |
+     * `-----------------------------------------------------------------------------------------'
+     */
+    [0] = LAYOUT_default(
```
Comment (noroadsleft) [inline keyboards/pursuit40/keymaps/via/keymap.c:36]:
```suggestion
    [0] = LAYOUT_all(
```

File: keyboards/pursuit40/keymaps/via/keymap.c [hunk 15]
```diff
@@ -0,0 +1,100 @@
...
+     * |   ~    | !    |  @   |  #  |  $   |  %   |  | ^  |  &   |  *  |  (    |  )  | BCKSPC  |
+     * |------------+------+------+------+------+-------------+------+------+------+------+----|
+     * |   \    |    1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  |   /   |
+     * |------------+------+------+------+------+------|------+------+------+------+------+----|
+     * |     |      |      |      |      |      |      |   +   |   =   |      |        |       |
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+----|
+     * |   ESC  |  CTRL-ALT-DEL   |   TASK   |         |        |    '|'    |    `    |        |
+     * `---------------------------------------------------------------------------------------'
+     */
+    [1] = LAYOUT_default(
```
Comment (noroadsleft) [inline keyboards/pursuit40/keymaps/via/keymap.c:55]:
```suggestion
    [1] = LAYOUT_all(
```

File: keyboards/pursuit40/keymaps/via/keymap.c [hunk 16]
```diff
@@ -0,0 +1,100 @@
...
+     * |       |      |      |  UP  |       |      |     |   _   |     |   [   |  ]    |      |
+     * |------------+------+------+------+------+-------------+------+------+------+------+---|
+     * |       |     | LEFT | DOWN  | RIGHT |     |      |   -  |      |   [   |   ]   |      |
+     * |------------+------+------+------+------+-----+-----+------+------+------+------+-----|
+     * |   F1  | F2 |  F3  |   F4  | F5    | F6  | F7   | F8   | F9    | F10  | F11   | F12   |
+     * |-------+-------+-------+-------+-------+-------+------+------+------+------+------+---|
+     * |     RESET    |           |          |         |         |         |        |         |
+     * `---------------------------------------------------------------------------------------'
+     */
+    [2] = LAYOUT_default(
```
Comment (noroadsleft) [inline keyboards/pursuit40/keymaps/via/keymap.c:74]:
```suggestion
    [2] = LAYOUT_all(
```

File: keyboards/pursuit40/keymaps/via/keymap.c (hunk 7 above)
Comment (noroadsleft) [inline keyboards/pursuit40/keymaps/via/keymap.c:93]:
```suggestion
    [3] = LAYOUT_all(
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]
GitHub tip: You can apply multiple suggestions to a single commit by using the [Files Changed tab](https://github.com/qmk/qmk_firmware/pull/11683/files).

**Review by npspears — COMMENTED** [review]
changes made

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

I usually lightly scold contributors for opening pull requests from modified `master` branches, but as you seem to reset yours after each of your PRs is merged, I'll let it slide. :)

**Pull request merged**